### PR TITLE
Stylistic fixes for clippy warnings

### PR DESCRIPTION
**Pull request by WorldSEnder**

### CODE REVIEW

File: src/command.rs [hunk 1]
```diff
@@ -1000,7 +998,11 @@ impl CommandBuffer {
             return Err(CommandError::TYPE_ERR);
         }
 
-        if !(RowMatrix::new(affine.transformation).det().abs() >= f32::EPSILON) {
+        if let Some(Ordering::Less) = RowMatrix::new(affine.transformation)
+            .det()
+            .abs()
+            .partial_cmp(&f32::EPSILON)
+        {
```
Comment (HeroicKatora) [inline src/command.rs:1005]: :nauseated_face: Also not the same. We want `NaN` to fail.

File: src/program.rs [hunk 2]
```diff
@@ -984,9 +984,9 @@ impl BufferInitContent {
     }
 
     /// Get a reference to the binary data, given the allocator/buffer.
-    pub fn as_slice<'lt>(&'lt self, buffer: &'lt Vec<u8>) -> &'lt [u8] {
+    pub fn as_slice<'lt>(&'lt self, buffer: &'lt [u8]) -> &'lt [u8] {
```
Comment (HeroicKatora) [inline src/program.rs:987]: No! `Vec<u8>` exists only because I was too lazy to make it a proper newtype. So this is a false positive.

**Review by HeroicKatora — COMMENTED** [review]
Somewhat surprisingly, only two instances of bikeshedding :joy:

File: src/command.rs (hunk 1 above)
Comment (WorldSEnder) [reply src/command.rs]: Next time I should just use the automatic rewriting... :robot:

**Pull request merged**